src/webui/www/private/index.html

```diff
@@ -1,9 +1,10 @@
-<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
-<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en" dir="ltr">
+<!DOCTYPE html>
+<html lang="${LANG}">
 <head>
     <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=10; IE=9; IE=8;" />
```

Comment:
Will the minimum supported version of IE now officially be 10?

Reply:
AFAIK, this directive is not for defining minimum supported version. It's for using legacy renderer in modern IE versions.
@Chocobo1?

Reply:
>Will the minimum supported version of IE now officially be 10?

IIRC it doesn't event work with IE version < 10.
So yes, kinda.

Reply:
In my testing it appears to work (mostly) well in IE9. IE8 doesn't work at all, which I'm rather ok with.

We ought to document this somewhere because it really does dictate what JaveScript features I can use when developing. Maybe on the qBittorrent [download](https://www.qbittorrent.org/download.php) page? @sledgehammer999